### PR TITLE
A lot of new & updated icons

### PR DESCRIPTION
**Pull request by Donnnno**
Here's the list:

**New**

- Fallout Shelter
- Flym
- ImgurViewer
- InstaGrabber
- Ma Livebox
- Metro
- Nebulo
- Syphon
- TrackerControl
- usageDirect
- Warden
- AIDS
- Notification CRON
- PicSayPro
- YetAnotherVideoPlayer
- Buienradar
- Our Home
- Pintrest
-  e.foundation: Apps (App store)
- yet another call blocker
- Flashy
- Pimple popper
- Ella
- Mirry
- Corona Melder
- Personal stickers for whatsapp
- System usage stats
- Happening
- Reddit Sync
- Fastreset
- Prey
- Radio France
- Greader Pro
- Mitch
- Dissembler

**Updated** (mostly removed circles )

- K9 Mail
- Metadataremover
- LibreOffice reader
- F-droid
- transportr
- Simple Gallery
- Slide for reddit
- Reroll
- Phonograph
- Scrambled exif
- Avans
- Letterboxd
- Pedometer
- WPS Office
- Snapseed
- Tasks
- Openscale
- Twidere
- Infinity

**These issues can be closed now:**

- #28
- #74
- #254
- #263
- #306
- #314
- #316
- #318
- #319
- #338
- #339 - empty request
- #347
- #348
- #349
- #350
- #354
- #358
- #360
- #365 
- ﻿#362
- #370

### CODE REVIEW

**Comment by Donnnno** [comment]
Updated some older icons, removed circles & some other small tweaks


image

**Comment by Donnnno** [comment]
Hi @dkanada, can you give me maybe a maintainer status or something like that?
Not for merging pull requests but more for closing icon requests & stuff to keep the project more organized :)

**Comment by dkanada** [comment]
Yeah I can look into it, I'll let you know :+1:

File: app/src/main/res/xml/appfilter.xml [hunk 1]
```diff
@@ -2238,9 +2328,6 @@
     <!-- Pocket Paint [aka Paintroid] -->
     <item component="ComponentInfo{org.catrobat.paintroid/org.catrobat.paintroid.MainActivity}" drawable="pocketpaint" />
 
-    <!-- Podomètre -->
-    <item component="ComponentInfo{de.j4velin.pedometer/de.j4velin.pedometer.ui.Activity_Main}" drawable="podomtre" />
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:2242]: Why was this one removed?

File: app/src/main/res/xml/appfilter.xml [hunk 2]
```diff
@@ -3713,6 +3829,11 @@
     <!-- Yatse -->
     <item component="ComponentInfo{org.leetzone.android.yatsewidgetfree/org.leetzone.android.yatsewidget.ui.StartActivity}" drawable="yatse" />
 
+    <!-- Yet Another Call Blocker --> <item component="ComponentInfo{dummydomain.yetanothercallblocker/dummydomain.yetanothercallblocker.MainActivity}" drawable="yetanothercallblocker"/>
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:3832]:
```suggestion
    <!-- Yet Another Call Blocker -->
    <item component="ComponentInfo{dummydomain.yetanothercallblocker/dummydomain.yetanothercallblocker.MainActivity}" drawable="yetanothercallblocker"/> 
```

**Comment by dkanada** [comment]
I checked the GitHub permissions, and I'd rather handle everything in this repository myself :sweat_smile: I prefer to avoid handing out permissions for repos on my personal account, but I also wanted to keep this app on my account so my GitHub profile doesn't look barren during job interviews.

**Comment by Donnnno** [comment]
Yeah, no worries! That's totally understandable :-)

**Pull request merged**